quickstarts/11-test-multi-agents/dapr-random.yaml

```diff
@@ -0,0 +1,24 @@
+# https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-template/#template-properties
+version: 1
+common:
+  resourcesPath: ./components
+  logLevel: info
+  appLogDestination: console
+  daprdLogDestination: console
+
+apps:
```

Comment:
add gandalf? shall we just combine this quickstart with the 05 one instead of adding a new quickstart?

Reply:
Yeah I was just trying to keep it separate to not mess with the other ones for now 😅.